src/components/Popover/set-activator-attributes.ts

```diff
@@ -0,0 +1,23 @@
+import {AriaAttributes} from 'react';
```

Comment:
It seems this chunk of code was never tested. When I was trying to add a few tests I kept getting stale dom nodes. I broke this utility into its own module so we can easily mock it in `Popover`.